sessions/redis.go

```diff
@@ -22,14 +22,22 @@ type RedisStore interface {
 //
 // It is recommended to use an authentication key with 32 or 64 bytes. The encryption key,
 // if set, must be either 16, 24, or 32 bytes to select AES-128, AES-192, or AES-256 modes.
-func NewRedisStore(size int, network, address, password string, keyPairs ...[]byte) (RedisStore, error) {
+func NewRediStore(size int, network, address, password string, DB string, keyPairs ...[]byte) (RedisStore, error) {
```

Comment:
changing the name of the func is not desirable because it will break the api for other users. Also I see that you add a new argument called `db` but you are not using it below at `redistore.NewRediStore`